Dockerfile.dapper

```diff
@@ -45,6 +45,12 @@ RUN apt-get -q update && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
     GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports
 
+RUN OP_FRAMEWORK="$GOPATH/src/github.com/operator-framework" && \
```

Comment:
Still hacky :P

We should make it more robust in the future, maybe stick a TODO here?

Reply:
ack, makes sense, I don't know why the guide instructions don't work here.

Reply:
Done